js/src/logger.ts

```diff
@@ -610,7 +637,7 @@ function spanComponentsToObjectIdLambda(
         "Impossible: computeObjectMetadataArgs not supported for experiments",
       );
     case SpanObjectTypeV2.PROJECT_LOGS:
-      const args = components.computeObjectMetadataArgs;
+      const args = { state, ...components.computeObjectMetadataArgs };
```

Comment:
nit: personally feels slightly clearer to have two arguments to `computeLoggerMetadata`, one being the exact struct that is serialized in span components, and another being the state.

Reply:
Sure